docs/topics/custom-images.md

```diff
@@ -0,0 +1,103 @@
+# Custom images
+
+This document will help you get a CAPZ Kubernetes cluster up and running with your custom image.
+
+## Reference images
+
+An *image* defines the operating system and Kubernetes components that will populate the disk of each node in your cluster.
+
+By default, images offered by "capi" in the [Azure Marketplace][azure-marketplace] are used. You can list these *reference images* with this command:
+
+```bash
+az vm image list --publisher cncf-upstream --offer capi --all -o table
+```
+
+But for more control over your nodes, you should build, publish, and use a *custom image*.
+
+## Building a custom image
+
+Cluster API uses the Kubernetes [Image Builder][image-builder] tools. You can use the [Azure image][image-builder-azure] from that project as a starting point for your custom image.
+
+[The Image Builder Book][capi-images] explains how to customize and build the images defined in that repository, with instructions for [Azure CAPI Images][azure-capi-images] in particular.
+
+### Operating system expectations
+
+TODO!
+
+### Required components
+
+TODO!
+
+## Publishing a custom image
+
+TODO!
```

Comment:
I don't know if that's something we want to direct users to do. Most users should use Shared Image Gallery to host their images as they won't need those images to be accessible to the public.